scripts/python/upload-with-submit.py

```
@@ -0,0 +1,239 @@
+        "Referer": "https://gemsofindia.org/submit",
+        "Sec-Ch-Ua": '"Chromium";v="140", "Not=A?Brand";v="24", "Google Chrome";v="140"',
+        "Sec-Ch-Ua-Mobile": "?0",
+        "Sec-Ch-Ua-Platform": '"macOS"',
+        "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) "
+                      "AppleWebKit/537.36 (KHTML, like Gecko) "
+                      "Chrome/140.0.0.0 Safari/537.36",
+        "Next-Action": "40c099e912187d2946439ebe43cc7e37e681f48caf",
+        "Next-Router-State-Tree": "%5B%22%22%2C%7B%22children%22%3A%5B%22 (entities)%22%2C%7B%22children%22%3A%5B%22submit%22%2C%7B%22children%22%3A%5B%22_PAGE_%22%2C%7B%7D%2Cnull%2Cnull%5D%7D%2Cnull%2Cnull%5D%7D%2Cnull%2Cnull%5D%7D%2Cnull%2Cnull%2Ctrue%5D",
+    }
+
+    r = requests.post(SUBMIT_URL, headers=headers, cookies=COOKIES, json=payload)
```

Ruff (0.14.4)

Probable use of `requests` call without timeout

(S113)